src/DataGrid.tsx

```diff
@@ -354,7 +354,10 @@ function DataGrid<R, SR>({
     scrollToRow(rowIdx: number) {
       const { current } = gridRef;
       if (!current) return;
-      current.scrollTop = rowIdx * rowHeight;
+      current.scrollTo({
```

Comment:
Does it work in safari 
https://caniuse.com/?search=scrollTo

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/ScrollToOptions
`ScrollToOptions` is supported, but the smooth option isn't supported yet so it'll be ignored.

>Last modified: Aug 21, 2019

The mdn page hasn't been updated for over a year so it's probably safe to use this.